### DISCUSSION TITLE
What is ubuntu version?

### DISCUSSION THREAD

**Discussion by MX-05** [timschneeb/GalaxyBudsClient#263] [Releases]
Hi, I don't know what file to download. I have ubuntu 20.04.3 LTS on a DELL notebook, what is the correct file to downoad the -Galaxy Buds client?

**Comment by timschneeb** [comment]
If your computer has a 64-bit processor, you need to download "GalaxyBudsClient_Linux_64bit_Portable.bin".

Before you can execute it, you need to give it proper permissions once in a terminal:

```
chmod +x GalaxyBudsClient_Linux_64bit_Portable.bin
```
Then execute it:
```
./GalaxyBudsClient_Linux_64bit_Portable.bin
```

**Reply by MX-05** [reply]
Thank you :-)

**Comment by n12iki** [comment]
@ThePBone 
Hi did something change, I cant seem to found the file you mentioned. I cloned the the repository. Sorry I am still getting to Ubuntu so I might have gotten something basic wrong.
Thanks

**Reply by timschneeb** [reply]
No, you can still find the file on the release page: https://github.com/ThePBone/GalaxyBudsClient/releases
Scroll down until you see a download link called `GalaxyBudsClient_Linux_64bit_Portable.bin`

**Reply by n12iki** [reply]
Awesome thank you. It looks amazing
